charts/kusion/templates/post-run-job.yaml

```diff
@@ -30,57 +31,20 @@ spec:
           command:
             - /bin/sh
             - -c
-            - >
```

Comment:
The backoffLimit reduction from 10 to 5 should be justified. Ensure this change won't cause premature job failures in case of temporary network issues.